<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formNouveauJeu</class>
 <widget class="QDialog" name="formNouveauJeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="CreerJeu">
       <property name="text">
        <string>Creer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AnnulerJeu">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nouveau Jeu :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="themeImage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom du nouveau Jeu :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nomJeu"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Image th√®me :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="theme"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="parcourirButton">
       <property name="text">
        <string>Parcourir</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Attributs</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listAttributSelect"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="ajouterButton">
           <property name="text">
            <string>&lt;--</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="retirerButton">
           <property name="text">
            <string>--&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listAttributDisp"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="nouveauAttributButton">
           <property name="text">
            <string>Nouveau</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modifierAttributButton">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AnnulerJeu</sender>
   <signal>clicked()</signal>
   <receiver>formNouveauJeu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
